app/controllers/withdraw_controller.rb

```diff
@@ -36,4 +43,11 @@ def new
       redirect_to document
     end
   end
+
+private
+
+  def user_has_permissions?
+    current_user.has_all_permissions?([User::MANAGING_EDITOR_PERMISSION,
+                                       User::PRE_RELEASE_FEATURES_PERMISSION])
```

Comment:
Does this method do anything if the user doesn't have the permissions it looks like it would return a boolean?

Reply:
I've changed the new action to render a 403 status code if the user doesn't have these permissions.

Reply:
The new action now raises an error if the user doesn't have these permissions.